### PR TITLE
Laser Pistol

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Laser pistol with new visuals to replace retro laser pistols. Functionally identical (except fits in pockets), but looks better imo.

## Why / Balance
I personally don't like the sprites for retro laser pistol, feel free to close this PR if you disagree

## How to test
Spawn LibertyVend
Buy Laser pistol

## Media
image
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
none afaik

**Changelog**
:cl: erhardsteinhauer
- tweak: Laser pistols have a new sprite, and take up a 1x2 rectangle in inventories.

### CODE REVIEW

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml
Comment (whatston3) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:111]:
```suggestion
  description: Slightly outdated design of civilian-grade laser pistol.
```

**Review by whatston3 — COMMENTED** [review]
Chunky gun, flashy lights, I like the design a lot.

One thing: is this supposed to be a complete replacement? Found three sources otherwise:
* ShuttleWeaponLaserGun still spawns the retro laser (weapon_cases_shuttle_gun.yml)
* Merc NPC tier 3 medics can spawn one (npc_loot_mercenaries.yml)
* CBURN units could spawn with them (in their pocket - once it comes out, it won't go back in - misc_startinggear.yml)

With respect to pocket loadout, seems strong, and a good backup option.  Only thing I could see being an issue is carrying a ton of laser pistols to cheese expeditions (should they have airlocks without glass more often?).  Don't really have any strong thoughts either way apart from that.

**Comment by ErhardSteinhauer** [comment]
> Only thing I could see being an issue is carrying a ton of laser pistols to cheese expeditions (should they have airlocks without glass more often?).

Players are already doing that with retro laser pistols. I don't believe that 2 additional pistols in their pockets will make a difference. I haven't changed inventory footprint of pistols.

> One thing: is this supposed to be a complete replacement?

Not really, but might as well do it.

**Review by whatston3 — APPROVED** [review]
Changes look good, love the new sprite, but I'd like a second opinion on balance.

**Comment by VividPups** [comment]
Retro shouldn't be replaced completely but be a unique rare weapon instead

**Comment by ErhardSteinhauer** [comment]
Nah, it should become a weapon for laser tag

**Comment by ErhardSteinhauer** [comment]
It already looks like a knock off Star Track toy, might as well go all the way and turn it into practice laser pistol.

**Comment by VividPups** [comment]
...that's the reference star trek

**Comment by ErhardSteinhauer** [comment]
Don't care. Looks like ass.

**Comment by ErhardSteinhauer** [comment]
Can't even tell where is the dangerous end of that pistol.

**Comment by dvir001** [comment]
Should get this in before upstream merge honestly.

**Pull request merged**
